srcpkgs/intel-parallel-studio-xe/template

```diff
@@ -0,0 +1,591 @@
+# Template file for 'intel-parallel-studio-xe'
+_remove_docs=true
+_remove_static_objects_mkl=false
+_remove_static_objects_ipp=false
+
+_year=2019
+_v_a=5
+_v_b=281
+_update=5
+_sp='cluster_edition'
+_icc_ver='19.0.5'
+_vtune_ver='2019.6.0.602217'
+_inspector_ver='2019.5.0.602103'
+_advisor_ver='2019.5.0.602216'
+_dir_nr=15809
+_parallel_studio_xe_dir="parallel_studio_xe${_year:+_${_year}}${_update:+_update${_update}}${_sp:+_${_sp}}"
+
+pkgname=intel-parallel-studio-xe
+version=${_year}.${_v_a}.${_v_b}
+revision=1
+wrksrc="${_parallel_studio_xe_dir}"
+archs="x86_64"
+repository="nonfree"
+short_desc="Intel C++ C and FORTRAN compiler"
+maintainer="Joseph Benden <joe@benden.us>"
+license="custom"
+homepage="https://intel.com/"
+distfiles="http://registrationcenter-download.intel.com/akdlm/irc_nas/tec/${_dir_nr}/${_parallel_studio_xe_dir}.tgz"
+checksum=c03421de616bd4e640ed25ce4103ec9c5c85768a940a5cb5bd1e97b45be33904
+subpackages="intel-common-libs intel-openmp intel-compiler-base intel-mpi intel-fortran-compiler intel-ipp intel-mkl intel-tbb-psxe intel-vtune-amplifier intel-vtune-amplifier-dkms intel-advisor intel-inspector"
```

Comment:
variable is only needed if not all packages specified if some subpackages are built conditionally, or order of creation matters, is it case here?